test/runtests.jl

```diff
@@ -48,39 +48,39 @@ let s = "a string チャネルパートナーの選択"
     close(p)
 end
 
-@test_throws ErrorException encode("qwertyé€", "ASCII")
+@test_throws InvalidSequenceError encode("qwertyé€", "ASCII")
 try
     encode("qwertyé€", "ASCII")
 catch err
      io = IOBuffer()
      showerror(io, err)
      @test takebuf_string(io) ==
-        "iconv error: byte sequence 0xc3a9e282ac is invalid in source encoding or cannot be represented in target encoding"
+        "iconv: byte sequence 0xc3a9e282ac is invalid in source encoding or cannot be represented in target encoding"
```

Comment:
Let's drop the mentions of iconv here and below: if you want to add Julia code for that, we'll print similar error messages and the caller shouldn't have to care about the implementation.